spec/models/question_advice_spec.rb

```
@@ -0,0 +1,118 @@
+
+    describe '#score' do
+        it 'returns the score of QuestionAdvice' do
+            expect(question_advice1.score).to eq(5)
+            expect(question_advice2.score).to eq(6)
+            expect(question_advice3.score).to eq(1)
+        end
+        
+    end
+
+    describe '#advice' do
+        it 'returns the score of QuestionAdvice' do
```

RuboCop (1.86.1)

[convention] Use 2 (not 4) spaces for indentation.

(Layout/IndentationWidth)